package.json

```diff
@@ -138,7 +137,8 @@
     "shelljs": "^0.8.5",
     "surge": "^0.21.5",
     "type-graphql": "^1.1.1",
-    "typechain": "^6.0.2"
+    "typechain": "^6.0.2",
+    "webpack-cli": "^4.9.2"
```

Comment:
Needed?

Reply:
It makes it easy to check what webpack version we are using. The issues with the dependency conflict was related to webpack versioning.

Reply:
Better to use resolutions to force a particular web pack version, then.

Reply:
A dev can always install webpack-cli globally if they want to use it, no? If you want to keep the web pack version consistent, then I would pin to a particular version using resolutions.